### PR TITLE
Bazel 0.28 support

### LINKED ISSUE #989: Bazel 0.28 support
Update rules_haskell to bazel 0.28.

### PR DESCRIPTION
**Pull request by guibou**
Straight forward upgrade. Just a few dependencies path where updated
to their latest versions.

Closes #989

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -372,16 +372,16 @@ skydoc_repositories()
...
+    urls = ["https://github.com/bazelbuild/rules_go/archive/0.18.7.tar.gz"],
 )
 
 http_archive(
     name = "com_github_bazelbuild_buildtools",
-    sha256 = "86592d703ecbe0c5cbb5139333a63268cf58d7efd2c459c8be8e69e77d135e29",
-    strip_prefix = "buildtools-0.26.0",
-    urls = ["https://github.com/bazelbuild/buildtools/archive/0.26.0.tar.gz"],
+    sha256 = "81fa031663fe67eab14b6b20131e5af1155f3e60b58da71249a3c744903710cd",
+    strip_prefix = "buildtools-0ce08111cb60d0f284a4399a36e23c28a934558a",
```
Comment (mboes) [inline WORKSPACE]: buildtools 0.28 just came out: https://github.com/bazelbuild/buildtools/releases/tag/0.28.0. Is that one sufficient?

Comment (guibou) [reply WORKSPACE]: Hey, good point.

Comment (guibou) [reply WORKSPACE]: Done.

**Comment by guibou** [comment]
Pushed the usage of `buildtools-0.28` as suggested by @m. I also realized that I did not had upgraded bazel for azure and ghc_bindist, so the result of the CI was mostly irrelevant ;) We'll see now that's done.

File: nixpkgs/default.nix [hunk 2]
```diff
@@ -1,4 +1,4 @@
 import (fetchTarball {
-   url = https://github.com/NixOS/nixpkgs/archive/a2075a2c314ccb3df6522dd957bb4e237446dc49.tar.gz;
-   sha256 = "1jbnjw06mca7867q81v3i3jljj0wpm38y9zjwij3nd6pp2bqhvbr";
+   url = https://github.com/guibou/nixpkgs/archive/f612ab79002c546d668c4f4ee3d5321fc814b838.tar.gz;
```
Comment (aherrmann) [inline nixpkgs/default.nix]: You mentioned the update is merged, can this point to upstream again?

Comment (guibou) [reply nixpkgs/default.nix]: I missed that. Sorry. This also explains the CI failure on nixpkgs.

Done. Waiting for CI.

**Pull request merged**